front/app/components/admin/PostManager/components/FilterSidebar/index.tsx

```diff
@@ -198,28 +189,60 @@ const FilterSidebar = ({
   const selectedItem = items.find((i) => i.key === activeFilterMenu);
   return (
     <>
-      <Menu
-        tabular
-        attached="top"
-        size="tiny"
+      <Box
+        display="flex"
+        flexDirection="row"
         className="intercom-admin-input-manager-filter-sidebar"
       >
-        {items.map((item) => (
-          <Menu.Item
-            key={item.key}
-            id={item.key}
-            active={activeFilterMenu === item.key}
-            onClick={handleItemClick}
-            className={`intercom-admin-input-manager-filter-sidebar-${item.key}`}
-            data-cy={`e2e-admin-post-manager-filter-sidebar-${item.key}`}
-          >
-            {item.name}
-          </Menu.Item>
-        ))}
-      </Menu>
-      <Segment attached="bottom">{selectedItem.content}</Segment>
+        {items.map((item) => {
+          const active = activeFilterMenu === item.key;
+
+          return (
+            <Box
+              display="flex"
+              flexDirection="row"
+              justifyContent="space-between"
+              w="100%"
+              as="button"
+              key={item.key}
+              id={item.key}
+              p="12px"
+              className={`intercom-admin-input-manager-filter-sidebar-${item.key}`}
+              data-cy={`e2e-admin-post-manager-filter-sidebar-${item.key}`}
+              {...getBorders(active)}
+              onClick={() => {
+                handleItemClick(item.key);
+              }}
+              style={{ cursor: 'pointer' }}
```

Comment:
Okay for me to merge without this but feel free to still take it up, as well as removing the bottom border of the tab that got selected (if effort is minor).

Reply:
> Wasn't it possible to use our Button component? The flexbox attributes can perhaps move to a wrapper between Button and item.name?

I don't think it's a good match for the button component. Maybe it has the cursor pointer by default, but it also has 10 other attributes (a background color, a border, a default padding, a default margin, hover effects...) which I all don't need so which I would have to disable.

I know people don't like it when `style` is used but I don't think it's such a big deal, and also it's much easier than to make a styled component just for this one attribute, and then you have to scroll back and forth because some of the attributes are defined in the styled component and others aren't.

Reply:
Actually I will just add the `cursor` property to the `Box`, seems like the best solution to me

Reply:
> I know people don't like it when style is used but I don't think it's such a big deal

It depends. For example, if we'd put it on a memoized component, memoization wouldn't work because the `style` object gets re-created on each re-render. And there are the reusability/trackability benefits of using props. 

Think the prop is a good call! 👍